lib/vector-buffers/Cargo.toml

```diff
@@ -25,7 +25,7 @@ metrics.workspace = true
 num-traits = { version = "0.2.19", default-features = false }
 paste = "1.0.15"
 pin-project.workspace = true
-rkyv = { version = "0.7.45", default-features = false, features = ["size_32", "std", "strict", "validation"] }
```

Comment:
* Always in strict mode since v0.8.
* `size_32` was renamed
* `validation` is no longer available

Reply:
Evaluating if this change is safe based on: https://github.com/rkyv/rkyv/releases/tag/0.8.0